app/controllers/users_controller.rb

```diff
@@ -1,7 +1,8 @@
 include Pagy::Backend
 class UsersController < ApplicationController
-  before_action :authenticate_user!
+  load_and_authorize_resource
   def index
+    add_breadcrumb t('shared.users'), :company_users_path
     @pagy, @users = pagy(User.where(params[:company_id]),  items: 5)
```

Comment:
```suggestion
    @pagy, @users = pagy(@users,  items: 5)
```
use the @users loaded by cancancan

Reply:
done